src/gui/activitywidget.cpp

```diff
@@ -174,55 +173,6 @@ void ActivityWidget::slotAccountActivityStatus(AccountState *ast, int statusCode
     showLabels();
 }
 
-// FIXME: Reused from protocol widget. Move over to utilities.
-QString ActivityWidget::timeString(QDateTime dt, QLocale::FormatType format) const
```

Comment:
This change appears to be unrelated cleanup and should be in a separate commit. Otherwise, the Git history becomes quite useless.

Reply:
I just removed custom date handling, we just provide a qdate to the view and it takes care of the presentation.
As we no longer sort by the display string we don't need to have seconds in the display string.